test/ocf/recovery/recovery-from-ram-wrong-disk.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+curdir=$(dirname $(readlink -f "${BASH_SOURCE[0]}"))
+rootdir=$(readlink -f $curdir/../../..)
+source $rootdir/test/ocf/common.sh
+source $rootdir/scripts/common.sh
+source $rootdir/test/common/autotest_common.sh
+rpc_py=$rootdir/scripts/rpc.py
+
+
+# Setup NVMe devices
+$rootdir/scripts/setup.sh
+
+# Create NVMe config
+prepare_nvme_config
+
+# Clear nvme device which we will use in test
+clear_nvme
+
+# Start SPDK app
+start_spdk "$curdir/config"
+
+# Create 2x256MiB partitions on NVMe device
+create_partitions Nvme0n1 2 256 
+
+# Create CAS device
+$rpc_py bdev_ocf_create cas_dev wb Nvme0n1p0 Nvme0n1p1 --create --force
+
+# Save config
+save_and_clean_bdev_config
+
+# Stop SPDK app
+stop_spdk
+
+# Copy original config file to temporaty file
+cp "$curdir/config" /tmp/config
+
+for i in 1 2
+do
+    # Clear NVMe devices
+    clear_nvme
+
+    # Change cache device in config
+    sed -i "s/Nvme0n1p$i/Nvme1n1/g" "$curdir/config"
```

Comment:
Might be a good idea to create 3 partitions on Nvme0 and use the third one here instead of Nvme1n1 (for setups with one NVMe drive)